tests/test_data/gen3_sample_input_standard_metadata.json

```diff
@@ -0,0 +1,218 @@
+[
```

Comment:
We should have a test with input that contains more than one bundle (e.g. two bundles)

Reply:
Agreed.